app/code/Magento/Checkout/Block/Cart/Totals.php

```diff
@@ -37,12 +37,18 @@ class Totals extends \Magento\Checkout\Block\Cart\AbstractCart
      * @var LayoutProcessorInterface[]
      */
     protected $layoutProcessors;
+    
+    /**
+     * @var \Magento\Checkout\Block\Cart\Helper\ConfigProvider
+     */
+    protected $cartConfig;
 
     /**
      * @param \Magento\Framework\View\Element\Template\Context $context
      * @param \Magento\Customer\Model\Session $customerSession
      * @param \Magento\Checkout\Model\Session $checkoutSession
      * @param \Magento\Sales\Model\Config $salesConfig
+     * @param \Magento\Checkout\Block\Cart\Helper\Config $cartConfig
```

Comment:
Please ensure the correct sequence of params in the phpdoc, it should correspond to the sequence of parameters in constructor